src/main/resources/org/sonar/plugins/checkstyle/rule-deletedChecks.xml

```diff
@@ -0,0 +1,2255 @@
+<rules>
```

Comment:
why do we need this file?

Reply:
It is for testing, whether `createMeta` matches default rule creator.

Reply:
Please provide more details on this

Reply:
Please provide more details on this

Reply:
Please provide more details on this

Reply:
To test whether the rule produced by the metadata utily matches the ones produced by default by the plugin, for every new check(those which are not present in `rule-deletedChecks.xml`), I have checked their properties against the original rule properties in `testCreate` UT. I hope I could explain.

Reply:
does this mean it's only for your proposal to know which checks you have verified and which you did not? then i'd either not commit it and keep it locally on your side or maybe movie it into a `todo` or similar directory naming it to `rules-unverified.xml`?

Reply:
No, it's not for that.  I wanted to test whether the new rules formed matches the old default rule properties or not. So first I created a `origRepo` with `rules.xml` and then a `modRepo` with `rule-deletedChecks.xml`. I forcefully removed 5 checks to trigger the creation of the new rules and checked the results in `testCreate` UT.
The `createMeta` creates rules for checks which are not present in the default repository formed from the xml file, thats why I created a  new file intentionally removing the checks.

Reply:
Please move this file to test resources